#goal definition
bool saveImgsForViz
string mapName
bool start
float32 mapStep
string sourceMap
---
#result definition
bool success
---
#feedback
